public/data/Pritam1812.json

```diff
@@ -0,0 +1,80 @@
+{
+    "name": "Pritam Hore",
+    "type": "personal",
+    "bio": "Aspiring Software Developer/Open Source Enthusiast!",
+    "avatar": "https://github.com/Pritam1812.png",
+    "links": [
+      {
+        "name": "Check me on GitHub!",
+        "url": "https://github.com/Pritam1812",
+        "icon": "github"
+      },
+      {
+        "name": "Connect with me on LinkedIn!",
+        "url": "https://www.linkedin.com/in/pritam1812",
+        "icon": "linkedin"
+      },
+      {
+        "name": "Check out my Website!",
+        "url": "https://pritam1812.github.io/PortfolioWebsite",
+        "icon": "globe"
+      },
+      {
+        "name": "Add me on Facebook!",
+        "url": "https://www.facebook.com/Pritam1812",
+        "icon": "Facebook"
+      },
+      {
+        "name": "Ping me on Instagram!",
+        "url": "https://instagram.com/__pritam.___",
+        "icon": "Instagram"
+      },
+      {
+        "name": "Find me on Twitter!",
+        "url": "https://twitter.com/_Pritam_1812",
+        "icon": "twitter"
+      },
+      {
+        "name": "Join me on Discord!",
+        "url": "https://discord.com/users/496729273776865282",

```

Comment:
I'm not sure if this will work properly. 😅
I'd recommend removing it or adding a proper invite to your server/community if you have. :)